autoscheduler/frontend/src/tests/redux/CourseCards.test.ts

```diff
@@ -274,4 +284,109 @@ describe('Course Cards Redux', () => {
       });
     });
   });
+
+  describe('deletes all course cards and resets numCardsCreated', () => {
+    test('when clearCourseCards is dispatched', () => {
+      // arrange
+      const store = createStore(autoSchedulerReducer, applyMiddleware(thunk));
+
+      // act
+      store.dispatch(clearCourseCards());
+
+      // assert
+      expect(store.getState().courseCards).toEqual({ numCardsCreated: 0 });
+    });
+  });
+
+  describe('replaces all course cards and keeps attributes not related to sections', () => {
```

Comment:
What does "keep attributes" mean here?

Reply:
This should probably say "properties", I mean all of the data for the card